epub33/rs/index.html

```diff
@@ -337,9 +337,10 @@ <h4>Metadata</h4>
 					<dt id="conf-metadata-link">The <code>link</code> element</dt>
 					<dd>
 						<p>Retrieval of Remote Resources is OPTIONAL.</p>
-						<p>The language identified in an <code>hreflang</code> attribute is not authoritative. Language
-							information defined in a linked resource determines its language.</p>
-						<p>Reading System do not have to use or present linked resources, even if they recognize the
+						<p>The language identified in an <code>hreflang</code> is purely advisory.
+							Upon fetching the resource, a Reading System must only use the language information associated with the resource to determine its language, not metadata included in the link to the resource.
+						</p>
+						<p>Reading Systems do not have to use or present linked resources, even if they recognize the
```

Comment:
Maybe go even simpler with: Support for linked resources is OPTIONAL. The rest is just filler.

Could probably be wrapped up with the statement about retrieval, too, since both are optional.

Reply:
Here is what I did in the latest commit:

- The first sentence now says "Retrieval and support of Remote Resources is OPTIONAL."
- I turned all the MUST-s and MAY-s into must-s and may-s (because the whole section is on an optional feature)

That being said, the proposal of @dlazin would read as:

> Reading Systems may use or present linked resources, if they recognize the relationship defined in the <code>rel</code> attribute.

which is slightly different; it does not say what the RS has to do if they do not recognize the relationship. I opted for simply commenting out (for now) the whole sentence, also as a consequence of making everything optional...

Reply:
Not sure I agree with this change. Wouldn't this mean that there are no normative requirements in media overlays because it's an optional feature? Supporting the feature and how you have to support it if you do are separate concerns.

Reply:
+1 to Matt's point. (I think he's referring to the "I turned all the MUST-s and MAY-s into must-s and may-s (because the whole section is on an optional feature)". I've seen other instances of MUSTs inside optional features.

No opinion on "Retrieval and support of Remote Resources is OPTIONAL." vs "Reading Systems may use or present linked resources, if they recognize the relationship defined in the rel attribute." I don't know enough about the implications.

Reply:
> Retrieval and support of Remote Resources is OPTIONAL.

This should be "linked resources". Linked resources are not necessarily remote resource (i.e., a linked resource can be located inside the container).

> Reading Systems may use or present linked resources

This statement is likely incomplete which is why I'm good with plain old "support". The reading system can do whatever it wants with linked resources. If it takes a metadata record and sends it to a downstream process, for example, is that really "using" it?

> if they recognize the relationship defined in the rel attribute

This kind of stands to reason. Also, by only talking about the relationships a reading system recognizes it seems to require another statement about the ones it doesn't. In either case, using the linked resources is optional. We can let devs sort out the ones they want to use.

Reply:
Something happened in the previous changes and commits, because the sentence on "Reading System may use or present..." sentence did not appear in it... And I think that is all right: the first sentence is, sort of, covering everything, and the reference to `rel` does not seem to add information. As @mattgarrish said, its usage stands to reason.

I have made some changes and, to avoid misunderstandings, this is how the text reads in the last iteration:

> **The link element**
>
> Retrieval and support of linked resources is OPTIONAL.
>
> The language identified in an `hreflang` is purely advisory. Upon fetching the resource, a Reading System must only use the language information associated with the resource to determine its language, not the metadata included in the link to the resource.
>
> In the case of a [linked metadata record](https://www.w3.org/TR/epub-33/#record) [EPUB-33], Reading Systems MUST NOT skip processing the metadata expressed in the Package Document and only use the information expressed in the record. Reading Systems may compile metadata from multiple linked records; they do not have to select only one record.
>
> When it comes to resolving discrepancies and conflicts between metadata expressed in the Package Document and in linked metadata records, Reading Systems MUST use the document order of `link` elements in the Package Document to establish precedence (i.e., metadata in the first linked record encountered has the highest precedence and metadata in the Package Document the lowest, regardless of whether the `link` elements occur before, within or after the package metadata elements).
>
 > Reading Systems MUST ignore any instructions contained in linked resources related to the layout and rendering of the EPUB Publication.

Are we o.k. now?

Reply:
> a Reading System must only use the language information associated with the resource

Shouldn't that one also be caps, or is that covered in the other issue?

Reply:
I guess it was a CAPS and that started this part of the issue:-)

I actually wonder whether we should not remove that part of the statement altogether. If the PR #1613 is merged, there is a separate section that covers all this language issues. The essence of that PR is that these types of statements can go.